requestfinder.py

```diff
@@ -199,10 +203,16 @@ def do_request(self, subcmd, opts):
 
         rqs = self.tool.find_requests(settings)
         for r in rqs:
-            if r.actions[0].type == 'submit':
-                print(' '.join(('#', r.reqid, r.actions[0].type, r.actions[0].src_project, r.actions[0].src_package, r.actions[0].tgt_project)))
-            else:
-                print(' '. join(('#', r.reqid, r.actions[0].type, r.actions[0].tgt_project)))
+            #if not r.actions[0].tgt_package.startswith('ghc-'):
```

Comment:
well, this really deserves a function. I can't spot any difference between the 2 code blocks you're touching